docs/notebooks/LORIS_API_Part2_Python-script.ipynb

```diff
@@ -0,0 +1,2159 @@
+   },
+   "outputs": [],
+   "source": [
+    "url = '/'.join([baseurl, 'candidates', candid_recs, visit_recs, 'recordings', \n",
+    "                recording_filename, 'electrodes', 'meta'])\n",
+    "candidateRecordingsElectrodesMeta = GET(url)\n",
+    "prettyPrint(candidateRecordingsChannels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "VjdEazWokc6O"
+   },
+   "source": [
+    "## Exercise 7.7 Get electrophysiology recording events data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "xF4ch6pQkdEj"
+   },
+   "outputs": [],
+   "source": [
+    "url = '/'.join([baseurl, 'candidates', candid_recs, visit_recs, \n",
+    "                'recordings', recording_filename, 'events'])\n",
+    "candidateRecordingsEvents = GET(url)\n",
+    "prettyPrint(candidateRecordingsEvents)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ROy3vUS7mSgv"
+   },
+   "source": [
+    "## Exercise 7.8 Get electrophysiology recording events meta data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "jKT8PA3kmTCR"
+   },
+   "outputs": [],
+   "source": [
+    "url = '/'.join([baseurl, 'candidates', candid_recs, visit_recs, 'recordings', \n",
+    "                recording_filename, 'events', 'meta'])\n",
+    "candidateRecordingsEventsMeta = GET(url)\n",
+    "prettyPrint(candidateRecordingsEventsMeta)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "HTJjrtTtZ_KN"
+   },
+   "source": [
+    "# Exercise 8 - Dicoms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "ZJhfo1iB_PqE"
+   },
+   "outputs": [],
+   "source": [
+    "# The flag is used to break the loops once an example that contains a \n",
+    "# Dicom file is found\n",
+    "flag = 0\n",
+    "\n",
+    "for candidate in allCandidates['Candidates']:\n",
+    "    candid = candidate['CandID']\n",
+    "    visit_labels = GET('/'.join([baseurl, 'candidates', candid]))['Visits']\n",
+    "    if flag == 1:\n",
+    "        break\n",
+    "    candid = candidate['CandID']\n",
+    "    visit_labels = GET('/'.join([baseurl, 'candidates', candid]))['Visits']\n",
+    "    for visit_label in visit_labels:\n",
+    "        dicoms = GET('/'.join([baseurl, 'candidates', candid, \n",
+    "                               visit_label, 'dicoms']))\n",
+    "        # We only want to get a single valid example of a candidate and visit that\n",
+    "        # has Dicom files that can be used to test the Dicoms endpoints \n",
+    "        if len(dicoms['DicomTars']) > 0:\n",
+    "            flag = 1\n",
+    "            break\n"
+   ]
+  },
```

Comment:
This block won't be necessary when PR #6775 will be merged. It will add an endpoint listing all Dicoms in a project. It will then be possible to get a complete list of candidates and for which visits there is a Dicom file for a given project. For the moment, it is the only way to get these informations without hardcoding them in the notebook, which is problematic when not executed with raisinbread...